Source/Core/GoogleEarthEnterpriseTerrainData.js

```diff
@@ -0,0 +1,521 @@
+         */
+        waterMask : {
+            get : function() {
+                return undefined;
+            }
+        }
+    });
+
+    var taskProcessor = new TaskProcessor('createVerticesFromGoogleEarthEnterpriseBuffer');
+
+    /**
+     * Creates a {@link TerrainMesh} from this terrain data.
+     *
+     * @private
+     *
+     * @param {TilingScheme} tilingScheme The tiling scheme to which this tile belongs.
+     * @param {Number} x The X coordinate of the tile for which to create the terrain data.
+     * @param {Number} y The Y coordinate of the tile for which to create the terrain data.
+     * @param {Number} level The level of the tile for which to create the terrain data.
+     * @param {Number} [exaggeration=1.0] The scale used to exaggerate the terrain.
+     * @returns {Promise.<TerrainMesh>|undefined} A promise for the terrain mesh, or undefined if too many
+     *          asynchronous mesh creations are already in progress and the operation should
+     *          be retried later.
+     */
+    GoogleEarthEnterpriseTerrainData.prototype.createMesh = function(tilingScheme, x, y, level, exaggeration) {
+        //>>includeStart('debug', pragmas.debug);
+        if (!defined(tilingScheme)) {
+            throw new DeveloperError('tilingScheme is required.');
+        }
+        if (!defined(x)) {
+            throw new DeveloperError('x is required.');
+        }
+        if (!defined(y)) {
+            throw new DeveloperError('y is required.');
+        }
+        if (!defined(level)) {
+            throw new DeveloperError('level is required.');
+        }
+        //>>includeEnd('debug');
+
+        var ellipsoid = tilingScheme.ellipsoid;
+        var nativeRectangle = tilingScheme.tileXYToNativeRectangle(x, y, level);
+        var rectangle = tilingScheme.tileXYToRectangle(x, y, level);
+        exaggeration = defaultValue(exaggeration, 1.0);
+
+        // Compute the center of the tile for RTC rendering.
+        var center = ellipsoid.cartographicToCartesian(Rectangle.center(rectangle));
+
+        var levelZeroMaxError = 40075.16; // From Google's Doc
+        var thisLevelMaxError = levelZeroMaxError / (1 << level);
+        this._skirtHeight = Math.min(thisLevelMaxError * 8.0, 1000.0);
+
+        var verticesPromise = taskProcessor.scheduleTask({
+            buffer : this._buffer,
+            nativeRectangle : nativeRectangle,
+            rectangle : rectangle,
+            relativeToCenter : center,
+            ellipsoid : ellipsoid,
+            skirtHeight : this._skirtHeight,
+            exaggeration : exaggeration,
+            includeWebMercatorT : true
+        });
+
+        if (!defined(verticesPromise)) {
+            // Postponed
+            return undefined;
+        }
+
+        var that = this;
+        return when(verticesPromise, function(result) {
+            that._mesh = new TerrainMesh(
+                center,
+                new Float32Array(result.vertices),
```

Comment:
I don't think so. It's passing in a buffer, not another typed array which should just use the `Float32Array` as a dataview into the buffer.